vocode/streaming/utils/events_manager.py

```diff
@@ -1,16 +1,22 @@
 from __future__ import annotations
 
 import asyncio
-from typing import List
+import os
+from typing import List, Optional
 
 from vocode.streaming.models.events import Event, EventType
 
 
 class EventsManager:
-    def __init__(self, subscriptions: List[EventType] = []):
+    def __init__(
+        self, subscriptions: List[EventType] = [], log_dir: Optional[str] = None
```

Comment:
it feels off to me to pass in a `log_dir` like this. it's not _really_ a first-class property of initializing an `EventsManager`. a centralized logging mechanism that `EventsManager` could leverage would probably be cleaner long-term, but fine for now

Reply:
Totally agree, we need real log collector (LogManager) for long term.